### PR TITLE
fixing tests

### PR DESCRIPTION
**Pull request by mm-binary**

### CODE REVIEW

File: MANIFEST [hunk 1]
```diff
@@ -9,6 +9,7 @@ lib/Quant/Framework/Utils/Types.pm
 LICENSE
 Makefile.PL
 MANIFEST			This list of files
+Quant-Framework-0.01.tar.gz
```
Comment (tfoertsch) [inline MANIFEST]: ???

File: Makefile.PL [hunk 2]
```diff
@@ -29,6 +28,7 @@ WriteMakefile(
         'Test::More' => 0,
         'Test::Exception' => 0,
         'Test::NoWarnings' => 0,
+        'Test::CheckManifest' => 1.29,
```
Comment (tfoertsch) [inline Makefile.PL]: You require this module but the related test has been removed. Did you move it anywhere else?

**Comment by tfoertsch** [comment]
I guess this is going to fail on cpantesters because of missing dependencies. But let's see.

**Comment by tfoertsch** [comment]
I think it's a good idea to add back the MANIFEST test because this time the manifest was wrong.

File: Makefile.PL (hunk 2 above)
Comment (mm-binary) [inline Makefile.PL]: Test is removed because its not used in other similar repos.

File: MANIFEST (hunk 1 above)
Comment (mm-binary) [inline MANIFEST]: Removed.

**Comment by mm-binary** [comment]
manifest checking is back and is passing on travis.

File: Makefile.PL [hunk 3]
```diff
@@ -16,7 +16,6 @@ WriteMakefile(
     AUTHOR           => q{Binary.com <support@binary.com>},
     VERSION_FROM     => 'lib/Quant/Framework.pm',
     ABSTRACT_FROM    => 'lib/Quant/Framework.pm',
-    LICENSE          => 'artistic_2',
     PL_FILES         => {},
     MIN_PERL_VERSION => 5.006,
```
Comment (tfoertsch) [inline Makefile.PL]: do we really care?

**Pull request merged**